### PR TITLE
feature: Polesia superclasses analysis

### PR DESCRIPTION
**Pull request by Crystallee1104**
Use pylandstat to analyse polesia superclass data

### CODE REVIEW

**Review by sdat2 — APPROVED** [review]
Looks good to me.

File: notebooks/exploratory/lyl-5-polesie-superclasses-pylandstat.ipynb [hunk 1]
```diff
@@ -0,0 +1,1836 @@
+{
```
Comment (KMacfarlaneGreen) [inline notebooks/exploratory/lyl-5-polesie-superclasses-pylandstat.ipynb:1]: <p>Just a small typo in 'Damaged bog'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai4er-cdt/gtc-biodiversity/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Review by KMacfarlaneGreen — DISMISSED** [review]
Looks great :)

**Review by herbiebradley — APPROVED** [review]
Nice work, this looks good!

**Pull request merged**